content/millennium/r4/individuals/related-person.md

```diff
@@ -109,13 +125,25 @@ List an individual RelatedPerson by its id:
 <%= headers status: 200 %>
 <%= json(:r4_relatedperson_entry) %>
 
+### Example - Encounter level
+
+#### Request
+
+    GET https://fhir-ehr-latest.cerner.com/r4/ec2458f2-1e24-41c8-b71b-0e701af7583d/RelatedPerson/E-12457994-97697434
+
+#### Response
+
+<%= headers status: 200 %>
+<%= json(:r4_relatedperson_encounter_entry) %>
+
 ### Errors
 
 The common [errors] and [OperationOutcomes] may be returned.
 
 [`CodeableConcept`]: https://hl7.org/fhir/r4/datatypes.html#CodeableConcept
-[`reference`]: https://hl7.org/fhir/R4/search.html#reference
+[`Reference`]: https://hl7.org/fhir/r4/search.html#reference
```

Comment:
Should this be uppercase or lowercase?

Reply:
Looks to be lowercase in most places.

Reply:
Updated: https://github.com/cerner/fhir.cerner.com/pull/526/commits/bb25db82b8811dbd47006c49a9139d3c0cfc824f#diff-e6b1787a35b755b7855af66f7e6243aa4d41f2e1b9c63544e414590074cf99abR145